engines/bastion_katello/app/assets/javascripts/bastion_katello/capsule-content/capsule-content.controller.js

```diff
@@ -86,8 +87,17 @@ angular.module('Bastion.capsule-content').controller('CapsuleContentController',
                     $scope.syncTask = aggregateTasks(syncStatus['active_sync_tasks']);
 
                 } else if (syncStatus['last_failed_sync_tasks'].length > 0) {
+                    errorCount = $scope.syncTask.humanized.errors.length;
                     $scope.syncTask = pickLastTask(syncStatus['last_failed_sync_tasks']);
-                    $scope.syncErrorMessages = $scope.syncTask.humanized.errors;
+                    if (errorCount > 0) {
+                        $scope.syncErrorMessages = [$scope.syncTask.humanized.errors[0]];
+                    }
+
+                    if (errorCount > 2) {
```

Comment:
The `else` would make it not display the "plus" string starting on line 97. Basically, if there are any errors at all, show the first one, then if there are any additional errors, show the message showing there are plural errors.